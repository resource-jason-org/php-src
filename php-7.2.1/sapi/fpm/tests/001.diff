001+ sh: 1: -n: not found
002+ NULL
001- string(%d) "PHP %s (fpm%s (built: %s
002- Copyright (c) 1997-20%s The PHP Group
003- Zend Engine v%s, Copyright (c) 1998-20%s Zend Technologies
004- "