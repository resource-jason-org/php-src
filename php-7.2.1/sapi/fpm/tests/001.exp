string(%d) "PHP %s (fpm%s (built: %s
Copyright (c) 1997-20%s The PHP Group
Zend Engine v%s, Copyright (c) 1998-20%s Zend Technologies
"
Done