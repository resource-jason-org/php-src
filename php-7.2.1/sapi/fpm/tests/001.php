<?php

include "include.inc";

$php = get_fpm_path();

var_dump(`$php -n -v`);

echo "Done\n";
?>
