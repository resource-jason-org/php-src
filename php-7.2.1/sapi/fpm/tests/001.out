sh: 1: -n: not found
NULL
Done